games-arcade/ceferino/ceferino-0.97.8-r2.ebuild

```
@@ -1,34 +1,34 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
+
 inherit autotools desktop
 
 DESCRIPTION="Super-Pang clone (destroy bouncing balloons with your grapnel)"
 HOMEPAGE="http://www.losersjuegos.com.ar/juegos/ceferino"
 SRC_URI="mirror://debian/pool/main/c/ceferino/${PN}_${PV}+svn37.orig.tar.gz"
+S="${WORKDIR}/${P}+svn37"
```

```suggestion
S+="+svn37"
```
It's not done often, but this is safe to do.

---

> It's not done often, but this is safe to do.

Need to correct myself here, while (currently) safe *with portage* it's actually not per PMS which allows for `S` to be undefined in global scope.
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-10900011.1

So please forget what I said here. I've reverted this in commit 3dd5de66b4400f7b37ace36c4cb006b65f20a739

---

Got it. Thanks for letting me know :)